programs/contractors/contractors-schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "array",
+  "description": "Array of contractor profiles",
+  "items": {
+    "type": "object",
+    "description": "Contractor profile",
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "Full name of the contractor"
+      },
+      "bio": {
+        "type": "string",
+        "description": "Biography of the contractor"
+      },
+      "email": {
+        "type": "string",
+        "format": "email",
+        "description": "E-mail address of the contractor"
+      },
+      "website": {
+        "type": "string",
+        "format": "url",
+        "description": "Website URL of the contractor"
+      },
+      "github": {
+        "type": "string",
+        "description": "GitHub profile URL of the contractor"
+      },
+      "twitter": {
```

Comment:
As much as I hate to say it, should this be "X" instead of "twitter"?

Reply:
Actually make it the Unicode value `𝕏` 😆

Reply:
Why not support both?  It's that [Robustness Principle](https://en.m.wikipedia.org/wiki/Robustness_principle) thing.

Reply:
Ah, fair enough. I adapted this from the ambassador schema/readme, which still mentions Twitter

Reply:
Done!